src/tutorials/lua_basics_lightbulb.md

```diff
@@ -365,398 +390,227 @@ switchTrigger.interactedEvent:Connect(OnSwitchInteraction)
-
-1. Right click on the **Point Light** in the Hierarchy and select **Enable Networking**.
-2. Right click on the **Point Light** in the Hierarchy again and select **Create New Template from This**
-3. Name the template ``LightTemplate``.
-
-![Create New Template](../img/LightBulb/LightTemplate.png){: .image-frame .center loading="lazy" }
-
-You should now be able to find your template, and drag out new copies of it from the **My Templates** section of the **Project Content** window.
-
-!!! warning "If you created the template without enabling networking, follow these steps to network it:"
-    1. Drag the template in into the Hierarchy (if it is not already there).
-    2. Right click it and select **Deinstance This Object**.
-    3. Right click it again and select **Enable Networking**.
-    4. Right click it again and select **Update Template From This**.
-    5. Delete the template from the Hierarchy.
-
-### Add the Template as a Custom Property of the Script
-
-1. Delete the **LightTemplate** from the Hierarchy. You don't need the light in the scene until a player turns on the light switch.
-2. Select the **LightToggleScript** in the Hierarchy and look at the **Properties** window.
-3. Look in the **Project Content** window for the **LightTemplate**.
-4. Drag the **LightTemplate** from Project Content into the Properties window of the **LightToggleScript**. This will add the template as a **Custom Property** of the script.
-
-!["Custom Properties"](../img/LightBulb/customAssetRefDrag.png "Custom Properties are added like a list here."){: .image-frame .center loading="lazy" }
-
-### Add the Custom Property Variable to the Script
-
-To be able to spawn the template, you will need a way to reference it in your script.
-
-!["Custom Properties"](../img/LightBulb/customPropRefs.png "Copy the list of custom property variables to use them in your scripts."){: .image-frame .center loading="lazy" }
-
-1. Copy the code generated in the script's **Properties**. You can right click and select **Select All** to highlight all the text in the box.
-2. Open the **LightToggleScript** by double clicking it in the **Hierarchy**.
-3. On the first line, paste the code that you copied. It will look something like this:
-
-```lua
-local propLightTemplate = script:GetCustomPropert("LightTemplate")
-```
-
-- `propLightTemplate` is the name of the variable. You can change this variable name, but will have to make sure that you change it in this tutorial every single place `propLightTemplate` is used.
-- `script:GetCustomProperty("LightTemplate")` tells the script to look for the script's custom property named **LightTemplate** which references the unique ID of the `LightTemplate` object in **Project Content**.
-
-### Spawn the Template
-
-Next you will tell the script to spawn `lightTemplate` when the player interacts with the switch.
+-- References to the objects we will be interacting with in the script.
```

Comment:
Change to script as subject, to take the "we" out : Reference to objects the script needs to interact with